integrated_channels/canvas/dev_docs/postman_collections/Canvas.postman_collection.json

```diff
@@ -0,0 +1,445 @@
+{
+	"info": {
+		"_postman_id": "1e52ea29-e08c-4247-879a-04bd5d82a713",
+		"name": "Canvas",
+		"description": "APIs for interacting with Canvas LMS",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
+	},
+	"item": [
+		{
+			"name": "GET oauth code",
+			"request": {
+				"method": "GET",
+				"header": [],
+				"url": {
+					"raw": "{{canvas_url}}/login/oauth2/auth?client_id=10000000000002&response_type=code&state=edx-canvas-user-state&redirect_uri=https://example.com/oauth_complete",
+					"host": [
+						"{{canvas_url}}"
+					],
+					"path": [
+						"login",
+						"oauth2",
+						"auth"
+					],
+					"query": [
+						{
+							"key": "client_id",
+							"value": "10000000000002"
+						},
+						{
+							"key": "response_type",
+							"value": "code"
+						},
+						{
+							"key": "state",
+							"value": "edx-canvas-user-state"
+						},
+						{
+							"key": "redirect_uri",
+							"value": "https://example.com/oauth_complete"
+						}
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "GET Oauth key",
+			"request": {
+				"method": "POST",
+				"header": [],
+				"body": {
+					"mode": "formdata",
+					"formdata": [
+						{
+							"key": "grant_type",
+							"value": "authorization_code",
+							"type": "text"
+						},
+						{
+							"key": "client_id",
+							"value": "10000000000002",
+							"type": "text"
+						},
+						{
+							"key": "client_secret",
+							"value": "bSxFhGIBmqluIMe7fIc0S6fNU75WtD7sB7r6T8euVwoYJ6eeNZIU0cfBS4VwjHwJ",
```

Comment:
Is there a way to actually make the template "blank"? ie it has the key client_secret, but the template is not storing its value.
I know these are just for development, but it still seems odd to commit these values.

Reply:
yah I do not know of a good way to do that, I think I could set the environment to none or sth in postman then export these.. but for this round I can just clear these out of the json

Reply:
actually I can and should set these to the variable instead of the value

Reply:
updated to use vars that can be sett in postman, instead of raw values